pkgs/build-support/build-fhs-userenv/env.nix

```diff
@@ -60,6 +60,9 @@ let
     export PATH="/run/wrappers/bin:/usr/bin:/usr/sbin:$PATH"
     export TZDIR='/etc/zoneinfo'
 
+    # XDG_DATA_DIRS is used by pressure-vessel (steam proton) and vulkan loaders to find the corresponding icd
```

Comment:
this isn't the bubblewrap file https://github.com/NixOS/nixpkgs/tree/master/pkgs/build-support/build-fhs-userenv-bubblewrap

Reply:
oops. May as well do both